tests/integration/call_run_within_script_with_autokeras.py

```diff
@@ -0,0 +1,33 @@
+"""
+Search for a good model for the
+[MNIST](https://keras.io/datasets/#mnist-database-of-handwritten-digits) dataset.
+"""
+
+from tensorflow.keras.datasets import mnist
+import tensorflow_cloud as tfc
+
+import autokeras as ak
+
+# Automated MirroredStrategy: chief config with multiple GPUs
+tfc.run(
+    entry_point=None,
+    distribution_strategy="auto",
+    chief_config=tfc.COMMON_MACHINE_CONFIGS["V100_1X"],
+    worker_count=0,
+    docker_base_image="haifengjin/autokeras:1.0.3",
+)
+# Prepare the dataset.
+(x_train, y_train), (x_test, y_test) = mnist.load_data()
+print(x_train.shape)  # (60000, 28, 28)
+print(y_train.shape)  # (60000,)
+print(y_train[:3])  # array([7, 2, 1], dtype=uint8)
+
+# Initialize the ImageClassifier.
+clf = ak.ImageClassifier(max_trials=2)
+# Search for the best model.
+clf.fit(x_train, y_train, epochs=10)
+# Evaluate on the testing data.
+print('Accuracy: {accuracy}'.format(
+    accuracy=clf.evaluate(x_test, y_test)))
+
+clf.export_model().save('model.h5')
```

Comment:
Should we provide a GCS bucket location instead of storing in local disk?

Reply:
What is the best practices for saving a model with TFCloud? Do we have any doc or example of it I can follow?

Reply:
Please see:
https://github.com/tensorflow/cloud/blob/master/tests/integration/call_run_on_script_with_keras_save_and_load.py
https://github.com/tensorflow/cloud/blob/master/tests/testdata/save_and_load.py